packages/devtools_app/lib/src/memory/memory_heap_tree_view.dart

```diff
@@ -354,23 +354,21 @@ class HeapTreeViewState extends State<HeapTree>
       );
     } else if (controller.snapshotByLibraryData != null ||
         controller.monitorAllocations.isNotEmpty) {
-      if (controller.showTreemap.value) {
-        snapshotDisplay = MemoryTreemap(controller);
-      } else {
-        snapshotDisplay = MemorySnapshotTable();
-      }
+      snapshotDisplay = MemoryHeapTable();
     } else {
+      // TODO: Have some help text about how to take a snapshot.
       snapshotDisplay = const SizedBox();
     }
 
     return Column(
       children: [
+        const SizedBox(height: denseRowSpacing),
```

Comment:
Nit: probably cleaner as padding on the column instead of a child of the column. That way the layout will still be what you want if some other tweaks to the Column's properties were made.

Reply:
done